.github/workflows/deploy-preview.yml

```
@@ -78,7 +78,7 @@ jobs:
                 owner: context.repo.owner,
                 repo: context.repo.repo,
                 issue_number: pullRequestNumber,
-                body: `${start} You can preview the tagging presets of this pull request [here](https://pr-${pullRequestNumber}--ideditor-presets-preview.netlify.app/id/dist/#locale=en).`
+                body: `${start} You can preview the tagging presets of this pull request [in this deploy preview](https://pr-${pullRequestNumber}--ideditor-presets-preview.netlify.app/id/dist/?id=n17807753#locale=en).`
```

The standalone version of iD does only look at the hash part of the url:

```suggestion
                body: `${start} You can preview the tagging presets of this pull request [in this deploy preview](https://pr-${pullRequestNumber}--ideditor-presets-preview.netlify.app/id/dist/#id=n17807753&locale=en).`
```

But for the  above mentioned alternatives, using the `map` parameter would be more appropriate:

```suggestion
                body: `${start} You can preview the tagging presets of this pull request [in this deploy preview](https://pr-${pullRequestNumber}--ideditor-presets-preview.netlify.app/id/dist/#locale=en&map=17.00/14.65485/121.06466).`
```